DuckDuckGo/SettingsViewModel.swift

```diff
@@ -553,13 +561,8 @@ extension SettingsViewModel {
 #if NETWORK_PROTECTION
         case .netP:
             if #available(iOS 15, *) {
-                switch NetworkProtectionAccessController().networkProtectionAccessType() {
-                case .inviteCodeInvited, .waitlistInvited:
-                    Pixel.fire(pixel: .privacyProVPNSettings)
-                    pushViewController(legacyViewProvider.netP)
-                default:
-                    pushViewController(legacyViewProvider.netPWaitlist)
-                }
+                Pixel.fire(pixel: .privacyProVPNSettings)
+                pushViewController(legacyViewProvider.netP)
```

Comment:
I confirmed with Gary directly that we're cool to remove the waitlist entry points, even for users without feature flag access. :+1: